src/app/TargetSelect/TargetSelect.tsx

```diff
@@ -85,9 +80,9 @@ export const TargetSelect: React.FunctionComponent<TargetSelectProps> = (props)
 
   const onSelect = (evt, selection, isPlaceholder) => {
     if (isPlaceholder) {
-      context.target.setTarget('');
+      context.target.setTarget({connectUrl:'', alias:''} as Target);
```

Comment:
This should also be refactored to use `NO_TARGET`, or at least some constant value defined within this class that is also used when initializing the state above.